packages/nodes-base/nodes/DateTime.node.json

```diff
@@ -11,5 +11,8 @@
 				"url": "https://docs.n8n.io/nodes/n8n-nodes-base.dateTime/"
 			}
 		]
-	}
+	},
+	"subcategories": [
+		"Helpers"
+	]
```

Comment:
@harshil1712 is it possible to update the structure to match the ticket?
https://n8n.atlassian.net/browse/PROD-1031

```
	"subategories": {
		"Core Nodes": ["Files"]
	}
```
for now your airtable and workflow only needs to support Core Nodes